### DISCUSSION TITLE
Welcome to OpenVoiceOS Discussions!

### DISCUSSION THREAD

**Discussion by ChanceNCounter** [OpenVoiceOS/ovos-buildroot#53] [Announcements]
## 👋 Hello from the OVOS team!
  This is a place for freeform discussion of OpenVoiceOS and related projects. If you have a question, suggestion, complaint, or just want to say hello, here's a great place to do it! You can also tell us about bugs here, if you aren't sure which repository to use.

  We can also be found in the primary OVOS thread at the MycroftAI forum. Individual developers can be reached in MycroftAI's official chat server. There is no OVOS channel on that server, but there is a HiveMind channel, where questions about crossover projects can be asked.

**Comment by bruce-a-williamson** [comment]
Just popping in to say hello. I had the Mark I code running on a RPI with the Google voice AIY kit. The mic volume was low so I set it aside. I came back to MyCroft and found out it was gone. Any idea of where to start up again?

**Comment by JarbasAl** [comment]
the easiest way to get started is probably with the ovos-installer, just install ovos on top of some existing OS (including raspbian)

https://github.com/OpenVoiceOS/ovos-installer

we also have a very active chat if you need help https://matrix.to/#/#openvoiceos-general:matrix.org
